### PR TITLE
Binary template docs and an extra example

### PR DESCRIPTION
**Pull request by kk7ds**
This adds docs for the template binary_sensor.

Also adds an automation example using a sensor and a notification, which seemed to be uncovered so far (took me a little fiddling to figure it out myself).

### CODE REVIEW

File: source/_cookbook/notify_if_over_threshold.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      entity_id: sensor.furnace
+      below: 2
+    action:
+      service: notify.mypushbullet
+      data:
+        title: "Furnace fan is stopped"
+        message: "Fan stopped because current is {% raw %}{{ states.sensor.furnace.state }}{% endraw %} amps"
+```
+
+Note that in this case, passing the state value through `round()` is required because state is a string and we need it to be converted to an integer before we compare it to another one. If you were comparing against an attribute that is already a number, then `round()` is not required.
```
Comment (balloob) [inline source/_cookbook/notify_if_over_threshold.markdown]: I don't see round being used in the examples

File: source/_components/binary_sensor.template.markdown [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+description: "Instructions how to integrate Template binary sensors into Home Assistant."
+date: 2016-02-25 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Binary Sensor
+---
+
+The `template` platform supports sensors which breaks out `state_attributes` from other entities.
```
Comment (balloob) [inline source/_components/binary_sensor.template.markdown]: Not just attributes, can be state too

Comment (kk7ds) [inline source/_components/binary_sensor.template.markdown]: I just copied this from sensor.template :)

**Comment by fabaff** [comment]
Thanks

**Pull request merged**